packages/embed-chart/src/App.tsx

```diff
@@ -58,7 +58,11 @@ function App(): JSX.Element {
           // Create the `ChartModel` for use with the `Chart` component
           log.debug(`Creating model...`);
 
-          const newModel = await ChartModelFactory.makeModel(undefined, figure);
+          const newModel = await ChartModelFactory.makeModel(
```

Comment:
Should use `const dh = useApi()` above, and not depend on `jsapi-shim`, instead pulling the types from `jsapi-types`.